frontend/src/BenchmarkTest.res

```diff
@@ -9,13 +9,61 @@ type testMetrics = {
 
 @module("../icons/branch.svg") external branchIcon: string = "default"
 
-let decodeMetricValue = (json): LineGraph.DataRow.value => {
+let isSize = (x) => Js.Re.exec_(%re("/(gb|mb|kb|bytes)\w*/i"), x)
+  ->Belt.Option.isSome
```

Comment:
Can we take it out to it's own file, something like "adjustMetricUnit" should be good :)